app/components/channel_loader/channel_loader.js

```diff
@@ -99,17 +100,27 @@ export default class ChannelLoader extends PureComponent {
     };
 
     render() {
-        const {channelIsLoading, maxRows, style: styleProp, theme} = this.props;
+        const {
+            channelIsLoading,
+            maxRows,
+            style: styleProp,
+            theme,
+        } = this.props;
 
         if (!channelIsLoading) {
             return null;
         }
 
         const style = getStyleSheet(theme);
         const bg = this.props.backgroundColor || theme.centerChannelBg;
+        const containerStyle = [style.container];
+
+        if (DeviceTypes.IS_TABLET) {
```

Comment:
I was surprised to see `DeviceTypes.IS_TABLET` -- which I presume is a constant -- vs. some run-time calculation that examines screen size. Intentional?

Reply:
Yes it is a constant defined here https://github.com/mattermost/mattermost-mobile/blob/master/app/constants/device.js#L21 and as you can see in the code it executes a function on the DeviceInfo object, this last object is actually a call to native that returns a boolean by checking the device type against a constant as the deviceType on be changing it makes sense to have it as a constant.

Android: https://github.com/mattermost/react-native-device-info/blob/master/android/src/main/java/com/learnium/RNDeviceInfo/RNDeviceModule.java#L140
iOS: https://github.com/mattermost/react-native-device-info/blob/master/ios/RNDeviceInfo/RNDeviceInfo.m#L276